yaas/parsers/bootstrap.py

```diff
@@ -0,0 +1,104 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import getpass
+import inspect
+
+from ..bootstrap import Bootstrap
+
+
+def command(parent_parser):
+    parser = parent_parser.add_parser(
+        'bootstrap',
+        help=inspect.getdoc(Bootstrap))
+
+    subcommands = [
+        bootstrap_agents,
+        bootstrap_show,
+        ]
+
+    subparsers = parser.add_subparsers()
+    for subcommand in subcommands:
+        subcommand(subparsers)
+
+
+def bootstrap_agents(parent_parser):
+
+    """ Install and configure ambari-agent on specified hosts. """
+
+    def agents(client, args):
+        with open(args.ssh_key, 'r') as f:
+            ssh_key = f.read()
+        res = client.bootstrap.agents(
+            hosts=args.hosts,
+            sshKey=ssh_key,
+            user=args.user,
+            userRunAs=args.userRunAs)
+        if not client.raw:
+            print(
+                'Request {id} {status} {log}'.format(
+                    id=res['requestId'],
+                    status=res['status'],
+                    log=res['log']))
+
+    parser = parent_parser.add_parser(
+        'agents',
```

Comment:
All of our yaas functions are singular. Because the endpoint supports specifying multiple agents it's okay to have a list for the bootstrap agent implementation function, but that should be transparent at the CLI.

yaas bootstrap agent foo bar goo